src/js/components/Menu/stories/ItemIcon.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Menu, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { FormDown, Github, Slack } from 'grommet-icons';
+
```

Comment:
Can we add a brief comment explaining why we added this story, perhaps with a link to the issue?